src/openrct2-ui/windows/TrackList.cpp

```diff
@@ -297,8 +297,7 @@ static Widget _trackListWidgets[] = {
                     }
                     break;
                 case WIDX_FILTER_STRING:
-                    WindowStartTextbox(
-                        *this, widgetIndex, STR_STRING, _filterString, sizeof(_filterString)); // TODO check this out
+                    WindowStartTextbox(*this, widgetIndex, _filterString, sizeof(_filterString)); // TODO check this out
```

Comment:
I think we should get rid of this TODO note. It does not convey any information.